docs/examples/webextension/background.js

```diff
@@ -0,0 +1,12 @@
+var TESTPILOT_TELEMETRY_CHANNEL = 'testpilot-telemetry';
+var testpilotPingChannel = new BroadcastChannel(TESTPILOT_TELEMETRY_CHANNEL);
+setInterval(function () {
+  testpilotPingChannel.postMessage({
+    boolData: true,
+    arrayOfData: ["one", "two", "three"],
+    nestedData: {
+      intData: 10,
+    },
+  });
```

Comment:
That is to say, what you send here is the `...` in the [example ping shown in README-METRICS.md for the `testpilottest` ping](https://github.com/mozilla/testpilot/blob/master/docs/README-METRICS.md#per-experiment-testpilottest-ping) - Test Pilot wraps that in the other envelope info and the full client environment data.